src/Controls/src/Core/Page/Page.cs

```diff
@@ -363,7 +367,7 @@ public Task<bool> DisplayAlert(string title, string message, string accept, stri
 		/// <param name="keyboard">The keyboard type to use for the user response.</param>
 		/// <param name="initialValue">A pre-defined response that will be displayed, and which can be edited by the user.</param>
 		/// <returns>A <see cref="Task"/> that displays a prompt display and returns the string value as entered by the user.</returns>
-		public Task<string> DisplayPromptAsync(string title, string message, string accept = "OK", string cancel = "Cancel", string placeholder = null, int maxLength = -1, Keyboard keyboard = default(Keyboard), string initialValue = "")
+		public Task<string> DisplayPrompt(string title, string message, string accept = "OK", string cancel = "Cancel", string placeholder = null, int maxLength = -1, Keyboard keyboard = default(Keyboard), string initialValue = "")
```

Comment:
`C:\a\_work\1\s\src\Controls\src\Core\Page\Page.cs(370,23): error CS0111: Type 'Page' already defines a member called 'DisplayPrompt' with the same parameter types [C:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net10.0-windows10.0.20348.0]`